apps/etterlatte-behandling/src/main/kotlin/sak/SakService.kt

```diff
@@ -90,12 +93,29 @@ class RealSakService(
                 enhet ?: finnEnhetForPersonOgTema(fnr, type.tema, type).enhetNr
             )
         }
+        this.sjekkSkjerming(fnr = fnr, sakId = sak.id)
         gradering?.let {
             tilgangService.oppdaterAdressebeskyttelse(sak.id, it)
         }
         return sak
     }
 
+    private fun sjekkSkjerming(fnr: String, sakId: Long) {
+        val erSkjermet = runBlocking {
+            skjermingKlient.personErSkjermet(fnr)
+        }
+        if (erSkjermet) {
+            inTransaction {
+                dao.oppdaterEnheterPaaSaker(
+                    listOf(GrunnlagsendringshendelseService.SakMedEnhet(sakId, Enheter.AALESUND.enhetNr))
+                )
+            }
+        }
+        inTransaction {
```

Comment:
Hvorfor er dette to transaksjoner? Ville trodd at det å endre enhet og markere som skjermet "hørte sammen" som en atomisk endring?

Reply:
Begynte å se på den før den ble merget :-)

Reply:
Ja men du har også de som skal markes som ikke skjermet lengre feks, og da skal vi ikke sette enhet=ÅLESUND på de.

Reply:
Men er i utgangspunktet enig at de hører sammen :)